docs/current_docs/quickstart/daggerize.mdx

```diff
@@ -55,7 +55,7 @@ can be inspected in Dagger Cloud.
 
 ### Initialize a Dagger module
 
-Bootstrap a new Dagger module in Go, Python or TypeScript by running `dagger init` in the application's root directory, using the `--source` flag to specify a directory for the module's source code.
+Bootstrap a new Dagger module in Go, Python, TypeScript or PHP by running `dagger init` in the application's root directory, using the `--source` flag to specify a directory for the module's source code.
```

Comment:
The tiniest nitpick you've probably ever experienced: 

This one uses _Typescript or PHP_

https://github.com/dagger/dagger/blob/23c2a97598249a0301c7ac32b41d0a89c99863f6/docs/current_docs/quickstart/daggerize.mdx?plain=1#L58

This one uses _Typescript, or PHP_

https://github.com/dagger/dagger/blob/23c2a97598249a0301c7ac32b41d0a89c99863f6/docs/current_docs/quickstart/index.mdx?plain=1#L17

I'm not fussed whether they both have a comma or don't but they should be consistent.

Reply:
Appreciate the detailed review! Fixed.